business_requirement_deliverable_project/models/business.py

```diff
@@ -43,6 +44,22 @@ def generate_projects_wizard(self):
         compute='_compute_planned_hour'
     )
 
+    @api.multi
+    def open_linked_projects(self):
```

Comment:
@darshan-serpent This returns type action so the name need to be according:
s/open_linked_projects
r/action_open_linked_projects